mock/py/mockchain.py

```diff
@@ -2,422 +2,7 @@
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA.
-# copyright 2012 Red Hat, Inc.
-
-# SUMMARY
-# mockchain
-# take a mock config and a series of srpms
-# rebuild them one at a time
-# adding each to a local repo
-# so they are available as build deps to next pkg being built
-from __future__ import print_function
-
-import cgi
-# pylint: disable=deprecated-module
-import optparse
-import os
-import re
-import shutil
-import subprocess
 import sys
-import tempfile
-import time
-
-import requests
-# pylint: disable=import-error
-from six.moves.urllib_parse import urlsplit
-
-import mockbuild.util
-
-# all of the variables below are substituted by the build system
-__VERSION__ = "unreleased_version"
-SYSCONFDIR = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), "..", "etc")
-PYTHONDIR = os.path.dirname(os.path.realpath(sys.argv[0]))
-PKGPYTHONDIR = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), "mockbuild")
-MOCKCONFDIR = os.path.join(SYSCONFDIR, "mock")
-# end build system subs
-
-mockconfig_path = '/etc/mock'
-
-
-def createrepo(path):
-    if os.path.exists(path + '/repodata/repomd.xml'):
-        comm = ['/usr/bin/createrepo_c', '--update', path]
-    else:
-        comm = ['/usr/bin/createrepo_c', path]
-    cmd = subprocess.Popen(
-        comm, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    out, err = cmd.communicate()
-    return out, err
-
-
-def parse_args(args):
-    parser = optparse.OptionParser('\nmockchain -r mockcfg pkg1 [pkg2] [pkg3]')
-    parser.add_option(
-        '-r', '--root', default=None, dest='chroot',
-        metavar="CONFIG",
-        help="chroot config name/base to use in the mock build")
-    parser.add_option(
-        '-l', '--localrepo', default=None,
-        help="local path for the local repo, defaults to making its own")
-    parser.add_option(
-        '-c', '--continue', default=False, action='store_true',
-        dest='cont',
-        help="if a pkg fails to build, continue to the next one")
-    parser.add_option(
-        '-a', '--addrepo', default=[], action='append',
-        dest='repos',
-        help="add these repo baseurls to the chroot's yum config")
-    parser.add_option(
-        '--recurse', default=False, action='store_true',
-        help="if more than one pkg and it fails to build, try to build the rest and come back to it")
-    parser.add_option(
-        '--log', default=None, dest='logfile',
-        help="log to the file named by this option, defaults to not logging")
-    parser.add_option(
-        '--tmp_prefix', default=None, dest='tmp_prefix',
-        help="tmp dir prefix - will default to username-pid if not specified")
-    parser.add_option(
-        '-m', '--mock-option', default=[], action='append',
-        dest='mock_option',
-        help="option to pass directly to mock")
```

Comment:
Except for this, the `mock --chain` should be compatible with `mockchain`, right?   Would you mind if I added small shell wrapper around `mock --chain`?